### PR TITLE
Website refresh with new logo, tagline, and pitch.

### PR DESCRIPTION
**Pull request by dherman**
Changes:
- New main page icon, favicon, and nav icon.
- New tagline: "Start your engines."
- New elevator pitch on main page.
- New color scheme.
- New font: ABeeZee
- Changed where we use the ⚡️ emoji on the main page.

[Updated preview](https://5e62d13b20f6d50009dd32b9--pensive-ardinghelli-58fa3b.netlify.com/)

~[Updated preview](https://5e62ce7cd3e134000a80a6c4--pensive-ardinghelli-58fa3b.netlify.com/)~

~[Updated preview](https://5e62c72a517f510008d99f22--pensive-ardinghelli-58fa3b.netlify.com/)~

~[Updated preview](https://5e62ab003a9f600007bd5950--pensive-ardinghelli-58fa3b.netlify.com/)~

~[Updated preview](https://5e6291913f34ac0008ba3d99--pensive-ardinghelli-58fa3b.netlify.com/)~

~[Preview](https://5e61fe95611208000b3a65fb--pensive-ardinghelli-58fa3b.netlify.com/)~

### CODE REVIEW

**Comment by dherman** [comment]
@chriskrycho I'd love a color accessibility review, even though I'm scared to learn that this scheme won't work 😬

**Comment by chriskrycho** [comment]
Giving it a look later this morning! And preemptively: usually if colors don’t work they can be tweaked while keeping the same feel!

File: theme/_sass/volta/_layout.scss [hunk 1]
```diff
@@ -151,35 +148,36 @@ main.home {
...
-      color: $color-primary;
+      color: $color-primary-dark;
     }
 
     .action-button {
       display: inline-block;
       font-size: 1.2rem;
       color: #fff;
-      background-color: $color-primary;
+      background-color: $color-primary-light;
```
Comment (chriskrycho) [inline theme/_sass/volta/_layout.scss]: This one needs to be a bit darker. If converted to HSL, 20% less "lightness" gets it there easily while keeping the look. Using `$color-primary` also works but ends up being fairly dark.

Happily, this is the only *contrast* issue!

**Comment by chriskrycho** [comment]
Update: that's going to be a bit more work than I originally thought – will require a small amount of restructuring DOM and CSS in a couple cases, not just adding `alt` attributes –so I'll try to do that in a *separate* PR. Those are also not *new* issues, though, so this PR doesn't cause regressions on that front.

File: theme/_sass/volta/_layout.scss (hunk 1 above)
Comment (dherman) [reply theme/_sass/volta/_layout.scss]: Thanks for all the tips on Discord today. I think we should be in good shape for color contrast now!

**Pull request merged**